### ISSUE TITLE
Hardware survey frequency on Linux (frequent rumor / possible bug)

### ISSUE THREAD

**Issue by qantourisc** [ValveSoftware/steam-for-linux#4349]
I have been reading several people that report less hardware surveys on Linux for years now.
There might be some truth to this rumour, and this might affect statistics (unless compensated for of course, or unless hardware Id's are tracked.)

It might be anecdotal, but one might wish to check it. I myself have completed more hardware surveys in Wine then I have in Linux.

Some quotes:

> I've been running Steam daily on Linux since 2013 and have received the survey exactly once. During the brief period where I used Steam through Wine (to play Batman and Deus Ex), I received it about 3 times, in the period of about two months, so yeah, you're not alone.

> I switched to Linux a year and a half ago, I use Steam almost everyday and not even once I've received a Steam Survey... 

> It seems that windows gets it a lot more than Linux, I got it many times on windows but this is the first on Linux.

**Comment by gdrewb-valve** [comment]
This is a duplicate of several open issues here, such as #2286.

**Comment by yanickrochon** [comment]
While the majority of those who commented on this issue (see the related issue), all issues about this have been closed without much explanation. It is a fact that the hardware and software survey *does not* appear on Linux as frequently than on other platforms. As I mentioned, I have Steam installed and running on a computer who's opened 24/7, and I got to complete the survey only once (or twice since I manually triggered it last winter). This has been confirmed by numerous users, but we are just being tossed aside. 

Is this how issues are resolved? Because the stats numbers shows a "decrease" of Linux users, yet users shout that the survey is not accurate, and every gaming engines make efforts to support Linux and Vulkan... it does not add up.
